data/mods/MindOverMatter/powers/electrokinesis.json

```diff
@@ -154,6 +154,45 @@
     "final_casting_time": 35,
     "casting_time_increment": -6
   },
+  {
+    "id": "electrokinetic_hacking_interface",
+    "type": "SPELL",
+    "name": "[Ψ]Hacking Interface (C)",
+    "description": "Create an electrical interface with your powers, allowing you to hack nearby devices as if you were using a computer.\n\nThis power <color_yellow>is maintained by concentration</color> and <color_red>may fail</color> if <color_yellow>concentration is interrupted</color>.",
+    "message": "",
+    "teachable": false,
+    "valid_targets": [ "self" ],
+    "spell_class": "ELECTROKINETIC",
+    "skill": "metaphysics",
+    "flags": [ "PSIONIC", "CONCENTRATE", "SILENT", "NO_HANDS", "NO_LEGS", "NO_EXPLOSION_SFX", "RANDOM_DURATION" ],
+    "difficulty": 4,
+    "max_level": { "math": [ "int_to_level(1)" ] },
+    "effect": "effect_on_condition",
+    "effect_str": "EOC_ELECTROKIN_HACKING_INTERFACE_INITIATE",
+    "shape": "blast",
+    "min_damage": 1,
+    "min_duration": {
+      "math": [
+        "min( (( ( (u_spell_level('electrokinetic_hacking_interface') * 4500 ) + 60000) * (scaling_factor(u_val('intelligence') ) ) ) * u_nether_attunement_power_scaling), 14400)"
```

Comment:
Did you mean to put a min on the duration here? As written, the maximum duration before a concentration check is only 2 minutes and 24 seconds (since the default 60000 + 4500 is already higher than 14400, it will always default to 14400)

Reply:
I see what you mean with the math and changed it to max.  I used the same duration logic as the personal battery power when creating this one. Would you like me to fix the personal battery power duration too?

Reply:
No thank you--that one is deliberate to make sure that the battery is silently swapped out in the background for a new one (with a fresh full charge) and that the power's duration doesn't outpace the electrokinetic's battery usage.